MantidQt/MantidWidgets/src/FitPropertyBrowser.cpp

```diff
@@ -73,19 +73,20 @@ FitPropertyBrowser::FitPropertyBrowser(QWidget *parent, QObject *mantidui)
       m_functionsGroup(nullptr), m_settingsGroup(nullptr),
       m_customSettingsGroup(nullptr), m_changeSlotsEnabled(false),
       m_guessOutputName(true),
+      m_fitMapper(nullptr), 
```

Comment:
Is it possible to declare the default nullptr inside the header, e.g.:
`QLabel *m_tip = nullptr;`

to avoid having this abomination of constructor in which the order is apparently wrong, and extremely confusing?